### PR TITLE
chore(tracing): wrap 'getBalanceForWallet' use-case in span

### PR DESCRIPTION
**Pull request by vindard**
## Description

Wrap `getBalanceForWallet` in a tracing span, related to debugging done in #793.

### CODE REVIEW

**Review by dolcalmi — APPROVED** [review]

File: src/app/wallets/get-balance-for-wallet.ts [hunk 1]
```diff
@@ -10,23 +11,31 @@ export const getBalanceForWallet = async ({
...
-      logger,
-    }),
-  ])
-  if (updatePaymentsResult instanceof Error) throw updatePaymentsResult
+}): Promise<Satoshis | ApplicationError> =>
+  asyncRunInSpan(
+    "app.getBalanceForWallet",
+    {
+      [SemanticAttributes.CODE_FUNCTION]: "getBalanceForWallet",
+      "enduser.walletId": walletId,
```
Comment (bodymindarts) [inline src/app/wallets/get-balance-for-wallet.ts:19]: This should be defined globally like [`ENDUSER_ALIAS`](https://github.com/GaloyMoney/galoy/blob/main/src/services/tracing.ts#L189).
Also it is semantically wrong as the enduser doesn't have a wallet - an account has wallets. Not a blocker as this adds value.

**Review by bodymindarts — APPROVED** [review]

**Review by nicolasburtey — APPROVED** [review]
we would benefits to override some methods (the resolver?) to avoid doing this for every use case

**Comment by vindard** [comment]
Waiting to see what comes from #813 for this PR

**Comment by nicolasburtey** [comment]
should we close this one now @vindard ?

**Comment by vindard** [comment]
Work done in #837

**Pull request closed without merging**